### PR TITLE
feat(ingestion/dbt): multiple node owner separated by comma

### PR DESCRIPTION
**Pull request by sid-acryl**

### CODE REVIEW

File: metadata-ingestion/src/datahub/utilities/mapping.py [hunk 1]
```diff
@@ -278,30 +302,48 @@ def get_operation_value(
             and operation_config[Constants.OWNER_TYPE]
         ):
             owner_id = _get_best_match(match, "owner")
+            owner_ids: List[str] = []
+
+            if "," in owner_id:
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/utilities/mapping.py:307]: we don't need the if statement here

File: metadata-ingestion/src/datahub/utilities/mapping.py [hunk 2]
```diff
@@ -186,22 +204,28 @@ def convert_to_aspects(
...
-            owner_aspect = OwnershipClass(
-                owners=[
+
+            owners: List[OwnerClass] = []
+
+            for x in sorted(
+                operation_map[Constants.ADD_OWNER_OPERATION],
+                key=lambda x: dict(x)["urn"],
+            ):
+                dict_owner_class = dict(x)
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/utilities/mapping.py:214]: why is this necessary? shouldn't `x` already be a dict?

File: metadata-ingestion/src/datahub/utilities/mapping.py [hunk 3]
```diff
@@ -160,7 +178,7 @@ def process(self, raw_props: Mapping[str, Any]) -> Dict[str, Any]:
                                 operation_type, set()
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/utilities/mapping.py:178]: let's add a special case for ownership, above the `if isinstance(operation, (str, list)):` line

operation_map[owner operation] should be a list, not a set.

File: metadata-ingestion/src/datahub/utilities/mapping.py (hunk 2 above)
Comment (sid-acryl) [reply metadata-ingestion/src/datahub/utilities/mapping.py:214]: operation_map[Constants.ADD_OWNER_OPERATION] is pointing to set so we do need to convert dict into tuple before storing it, hence x is tuple

File: metadata-ingestion/src/datahub/utilities/mapping.py (hunk 1 above)
Comment (sid-acryl) [reply metadata-ingestion/src/datahub/utilities/mapping.py]: ok

File: metadata-ingestion/src/datahub/utilities/mapping.py [hunk 4]
```diff
@@ -278,30 +302,48 @@ def get_operation_value(
...
-                    "category": owner_category,
-                    "categoryUrn": owner_category_urn,
-                }
-            elif operation_config[Constants.OWNER_TYPE] == Constants.GROUP_OWNER:
-                return {
-                    "urn": mce_builder.make_owner_urn(owner_id, OwnerType.GROUP),
-                    "category": owner_category,
-                    "categoryUrn": owner_category_urn,
-                }
+                sanitize_owner_ids = [
```
[the remark this answers is not in the document]
Comment (sid-acryl) [reply metadata-ingestion/src/datahub/utilities/mapping.py]: used it directly. I used a separate one for readability.

File: metadata-ingestion/src/datahub/utilities/mapping.py (hunk 2 above)
Comment (sid-acryl) [reply metadata-ingestion/src/datahub/utilities/mapping.py]: added extra if condition, now it is dict only

File: metadata-ingestion/src/datahub/utilities/mapping.py (hunk 3 above)
Comment (sid-acryl) [reply metadata-ingestion/src/datahub/utilities/mapping.py:190]: done

**Review by hsheth2 — APPROVED** [review]
